pkgs/applications/editors/neovim/wrapper.nix

```diff
@@ -68,28 +71,34 @@ let
   in
   assert withPython2 -> throw "Python2 support has been removed from the neovim wrapper, please remove withPython2 and python2Env.";
 
-  symlinkJoin {
+  stdenv.mkDerivation (finalAttrs: {
       name = "neovim-${lib.getVersion neovim}${extraName}";
+
+      __structuredAttrs = true;
+      unpackPhase = ":";
+      inherit viAlias vimAlias withNodeJs withPython3;
+      inherit providerLuaRc packpathDirs;
+
```

Comment:
```suggestion
      nativeBuildInputs = [ lndir ];
```

Reply:
I mean I ran those commands on master first, to make sure they worked there before running them on my changes. I could not make any cross compilation work on master so I can't test this PR (haven't even tried).

Reply:
Yep, neovim cross-compilation is broken but i have found a fix

Reply:
https://github.com/NixOS/nixpkgs/pull/244252#